claims_hosp/delphi_claims_hosp/run.py

```diff
@@ -54,19 +56,27 @@ def run_module(params):
                 adjustments (False).
     """
     start_time = time.time()
-    logger = get_structured_logger(
-        __name__, filename=params["common"].get("log_filename"),
-        log_exceptions=params["common"].get("log_exceptions", True))
+    issue_date_str = params.get("patch", {}).get("current_issue", None)
+    issue_date = datetime.strptime(issue_date_str, "%Y-%m-%d")
+    # safety check for patch parameters exists in file, but not running custom runs/patches
+    custom_run_flag = (
+        False if not params["indicator"].get("custom_run", False) else params["indicator"].get("custom_run", False)
+    )
+    if not logger:
```

Comment:
Should there be "if logger:" ?

Reply:
Nope, basically how the patching code has worked is that it's a wrapper to called the run script within in a for loop with some customization and in order to make sure that logging is different from patching and a regular run is to pass on a logger as a parameter that's created from patch.

if it's a regular run, it's not going to have that

So the logic goes, if the logger exists already, then it's logger from patch, if not we need to create the logger